src/ontology/edits/oeo-tasks.omn

```diff
@@ -0,0 +1,726 @@
+    
+    
+Class: OEO_00390049
+
+    Annotations: 
+        OEO_00020426 "Add;
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine wind turbine lifetime is an action specification that describes the process of estimating the operational lifespan of a wind turbine, based on design and site-specific conditions.",
+        rdfs:label "determine wind turbine lifetime"@en
+    
+    SubClassOf: 
+        OEO_00390048
+    
+    
+Class: OEO_00390050
+
+    Annotations: 
+        OEO_00020426 "Add;
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine capital expenditures is an action specification that describes the calculation of capital expenditures (CAPEX) required for the installation of wind turbines and associated infrastructure.",
+        rdfs:label "determine capital expenditures"@en
+    
+    SubClassOf: 
+        OEO_00390048
+    
+    
+Class: OEO_00390051
+
+    Annotations: 
+        OEO_00020426 "Add;
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine operational expenditures is an action specification that describes the estimation of ongoing operational expenditures (OPEX) for wind turbines, including routine maintenance and operational management. These include operating and maintenance costs.",
+        rdfs:label "determine operational expenditures"@en
+    
+    SubClassOf: 
+        OEO_00390048
+    
+    
+Class: OEO_00390052
+
+    Annotations: 
+        OEO_00020426 "Add;
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Determine discount and interest rate is an action specification that describes the selection of financial discount and interest rates, which are used in the economic analysis to account for the time value of money.",
+        rdfs:label "determine discount and interest rate"@en
+    
+    SubClassOf: 
+        OEO_00390048
+    
+    
+Class: OEO_00390053
+
+    Annotations: 
+        OEO_00020426 "Add;
+issue: https://github.com/OpenEnergyPlatform/ontology/issues/1891
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1940",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Calculate Levelized Cost of Energy is an action specification that describes the computation of the Levelized Cost of Energy (LCOE) either for individual turbines or for sub-regions within the study area by applying a cost model based on the previously calculated economic parameters and the capacity factors.",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000115> "Calculate levelised cost of electricity is an action specification that describes the computation of the levelised cost of electricity (LCOE) either for individual turbines or for sub-regions within the study area by applying a cost model based on the previously calculated economic parameters and the capacity factors.",
```
We use BE spelling.
LCOE is already a concept in OEO OEO_00020127, ToDo: add axiom later. There, the label is levelised cost of _electricity_. Since wind power refers to electrical energy, there's no harm in changing the label here.